### PR TITLE
CSS changes

### PR DESCRIPTION
**Pull request by josh-cook**
This PR makes changes to the CSS Layout using grid and also changes for removing the room (blue womb) that wasn't used.

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -11,15 +11,33 @@
   </head>
 
   <body>
-    <div id="pageTitle">
-      <h1>Binding Of Isaac: Challenge Generator</h1>
+    <div class="wrapper">
+      <div id="pageTitle">
+        <h1>Binding Of Isaac: Challenge Generator</h1>
+      </div>
+      <div class="challengeTitle"><span id="title"></span></div>
```
Comment (jackwilsdon) [inline html/index.html]: Is there a reason there is a span with an ID inside this div? Would this not be simpler?

```suggestion
      <div class="challengeTitle" id="title"></div>
```

File: html/index.html [hunk 2]
```diff
@@ -11,15 +11,33 @@
...
+      <div class="container">
+        <div>
+          Challenge #:<br />
+          <span id="seed"></span>
```
Comment (jackwilsdon) [inline html/index.html]: Could you make the value spans `display: block` so that the `<br />` on the previous line isn't needed?

**Review by jackwilsdon — APPROVED** [review]

File: html/index.html (hunk 2 above)
Comment (josh-cook) [reply html/index.html]: Really good idea, thanks

**Comment by jackwilsdon** [comment]
LGTM :+1:

**Pull request merged**